src/styles/card.css

```diff
@@ -0,0 +1,85 @@
+
+.card {
+  padding: 16px;
+  width: 166px;
```

Comment:
what's that width? it should be 200px, use `box-sizing: border-box` to avoid using these weird values